source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -246,23 +248,43 @@ def _extend_with_car_park(
     dp_journey.distances.walking += int(walking_speed * car_park_crowfly_duration)
 
 
-def append_path_item_with_access_point(path_items, stop_point, access_point):
+def append_path_item_with_access_point(path_items, stop_point, access_point, language):
+    def get_instruction_words(lan):
+        instructions = {
+            "english_us": u"Then Enter {} via {}.",
+            "french": u"Accéder à {} via {}.",
+            "spanish": u"acceso a {} vía {}.",
+            "italian": u"accesso alla {} vía {}.",
```

Comment:
No upper case for the first letter?

```suggestion
            "spanish": u"Acceso a {} vía {}.",
            "italian": u"Accesso alla {} vía {}.",
```

Reply:
:+1: